spec/lib/declare_schema/migration_generator_spec.rb

```diff
@@ -682,5 +688,84 @@ class Ad < ActiveRecord::Base
     ActiveRecord::Migration.class_eval(up)
     expect(Ad.field_specs['company'].options[:validates].inspect).to eq("{:presence=>true, :uniqueness=>{:case_sensitive=>false}}")
   end
-end
 
+  describe 'belongs_to' do
```

Comment:
This shows how broken `rdoctests` were! Since I TDD'd this, I wound up with 8-10 separate tests.